gmat/tests/build_frames_scripts.py

```diff
@@ -0,0 +1,86 @@
+from itertools import product
+
+from astropy import units as u
+from jinja2 import Template
+from numpy.linalg import norm
+from poliastro.bodies import Jupiter as Jupiter_poliastro
+from poliastro.bodies import Mars as Mars_poliastro
+from poliastro.bodies import Mercury as Mercury_poliastro
+from poliastro.bodies import Moon as Moon_poliastro
+from poliastro.bodies import Neptune as Neptune_poliastro
+from poliastro.bodies import Saturn as Saturn_poliastro
+from poliastro.bodies import Sun as Sun_poliastro
+from poliastro.bodies import Uranus as Uranus_poliastro
+from poliastro.bodies import Venus as Venus_poliastro
+
+# All interesting 3D directions
+R_SET, V_SET = [list(product([-1, 0, 1], repeat=3)) for _ in range(2)]
+
+# Name of the bodies
+BODIES_NAMES = [
+    "Sun",
+    "Mercury",
+    "Venus",
+    "Moon",
+    "Mars",
+    "Jupiter",
+    "Saturn",
+    "Uranus",
+    "Neptune",
+]
+
+POLIASTRO_BODIES = [
+    Sun_poliastro,
+    Mercury_poliastro,
+    Venus_poliastro,
+    Moon_poliastro,
+    Mars_poliastro,
+    Jupiter_poliastro,
+    Saturn_poliastro,
+    Uranus_poliastro,
+    Neptune_poliastro,
+]
+
+BODY_FROM_NAME = dict(zip(BODIES_NAMES, POLIASTRO_BODIES))
+
+if __name__ == "__main__":
+
+    # Open the template
+    with open("template_frames.script") as template_file:
+        template_raw = Template(template_file.read())
```

Comment:
After we read `template_file`, do we need all the rest to be inside the `with open(...)` block? I think we can move it outside

Reply:
Nope, you are right. Fixing indentation...